kuma/landing/views.py

```diff
@@ -8,8 +8,7 @@
 from kuma.core.sections import SECTION_USAGE
 from kuma.core.cache import memcache
 from kuma.feeder.models import Bundle
-from kuma.search.models import Filter, FilterGroup
-from kuma.search.serializers import GroupWithFiltersSerializer
```

Comment:
The ``GroupWithFiltersSerializer`` should also be deleted from ``kuma/search/serializers.py``

Reply:
I see `FilterWithGroupSerializer` in test_serializers.py as well.